build.rs

```diff
@@ -759,7 +759,7 @@ fn get_command(var: &'static str, default: &str) -> OsString {
 // environment variables that affect the build.
 fn emit_rerun_if_changed() {
     for path in &["crypto", "include", "third_party/fiat"] {
-        walk_dir(&PathBuf::from(path), &|entry| {
+        walk_dir(&absolute_path_from_manifest_dir_relative_path(path), &|entry| {
```

Comment:
I don't think this needs to be modified since any interpreter of the `cargo:rerun-if-changed` output would need to be able to handle relative paths. I would expect that Bazel wouldn't process the `cargo:rerun-if-changed` output anyway.